src/main/kotlin/com/github/markoresic/footballvotingdemo/auth/RegisterRequest.kt

```diff
@@ -1,10 +1,14 @@
 package com.github.markoresic.footballvotingdemo.auth
 
-import com.github.markoresic.footballvotingdemo.model.user.Role
+import jakarta.validation.constraints.Email
+import jakarta.validation.constraints.NotBlank
 
 data class RegisterRequest(
+    @field:NotBlank(message = "Email must not be blank")
+    @field:Email(message = "Email must be valid")
     val email: String,
+    @field:NotBlank(message = "Password must not be blank")
```

Comment:
What if you want to implement custom logic for password, let's say one special char and longer then 8? can you modify this?